spec/controllers/orders_controller_spec.rb

```diff
@@ -5,4 +5,26 @@
     it "returns http success" do
     end
   end
+
+  describe "POST #paypal hook" do
+    let(:order) do
+      create :order,
+             user: create(:regular_user),
+             address: create(:address)
+    end
+
+    before do
+      allow_any_instance_of(Order).to receive(:paypal_url).and_return(
+        status: 200,
+        body: "VERIFIED",
+        headers: {}
+      )
+    end
+
+    it "raises error if payment was not successful" do
```

Comment:
What about the test case for when payment is successful?
This could be a context say `context "payment not successful" `